src/adapters/sqliteAdapter.ts

```diff
@@ -33,7 +33,7 @@ export async function teardownDatabase(adapterOptions: EnvironmentAdapterOptions
     prismaFolder = customPrismaFolder.join('/')
   }
 
-  const databaseFile = resolve(process.cwd(), prismaFolder, `${databaseName.replace('.db', '')}_${databaseSchema}.db`)
+  const databaseFile = resolve(process.cwd(), `${databaseName.replace('.db', '')}_${databaseSchema}.db`)
```

Comment:
Why would you remove the `prismaFolder` path? It is used when the database schema is not at the root folder

Reply:
Sorry, that was a mistake when I made this commit. I'll clean it up in my next update.

Reply:
Actually, in this approach, the ephemeral database is gonna be created on the /tmp dir of the OS, so I think we don't need to specify the custom schema path. I think that the prisma script will find it by itself. I will check this implementation to see if it works on a few scenarios